src/main/kotlin/com/bugsnag/android/gradle/BugsnagPlugin.kt

```diff
@@ -267,18 +267,35 @@ class BugsnagPlugin : Plugin<Project> {
             val taskName = computeManifestTaskNameFor(output.name)
             val manifestInfoOutputFile = project.computeManifestInfoOutputV1(output)
             val buildUuidProvider = project.newUuidProvider()
-            project.tasks.register(taskName, BugsnagManifestUuidTask::class.java) {
+
+            val manifestTask = project.tasks.register(taskName, BugsnagManifestUuidTask::class.java) {
                 it.buildUuid.set(buildUuidProvider)
                 it.variantOutput = output
                 it.variant = variant
                 it.manifestInfoProvider.set(manifestInfoOutputFile)
-                val processManifest = output.processManifestProvider.orNull
+                it.dependsOn(output.processManifestProvider)
+                it.mustRunAfter(output.processManifestProvider)
```

Comment:
just curious, this line is not needed and doesn't change anything, right? 

> Gradle ensures that all task dependencies and ordering rules are honored when executing tasks, so that the task is executed after all of its dependencies

Reply:
Strictly speaking I don't believe it's required, but I think it's helpful making it very explicit here that the task must run after the `processManifest` task.